core/src/banking_stage/packet_filter.rs

```diff
@@ -1,10 +1,19 @@
 use {
     super::immutable_deserialized_packet::ImmutableDeserializedPacket,
-    solana_builtins_default_costs::BUILTIN_INSTRUCTION_COSTS,
-    solana_sdk::{ed25519_program, saturating_add_assign, secp256k1_program},
+    lazy_static::lazy_static,
+    solana_builtins_default_costs::get_builtin_instruction_cost,
+    solana_sdk::{
+        ed25519_program, feature_set::FeatureSet, saturating_add_assign, secp256k1_program,
+    },
     thiserror::Error,
 };
 
+lazy_static! {
+    // To calculate the static_builtin_cost_sum conservatively, an all-enabled dummy feature_set
+    // is used. It lowers required minimal compute_unit_limit, aligns with future versions.
+    static ref FEATURE_SET: FeatureSet = FeatureSet::all_enabled();
```

Comment:
why can't we just resolve the feature gate status at a higher level and use a bool instead of some poorly named sentinel value?

Reply:
Is this important for backport review? Also can you explain the bool you're talking about? I'm not getting what you're suggesting

Reply:
At the receiving & buffer part of Banking_stage, resolving feature gate status is always bit fuzzy since we have a 'working_bank' yet. There are certainly ways around it, but nothing is clean and easy afaik.